### ISSUE TITLE
It would be cool if this had a license

### ISSUE THREAD

**Issue by myyk** [Quillraven/SimpleKtxGame#15]
This is a cool demo and it's nice that the LibGDX docs link to these assets. I think it would be nice if there were a license so that it's clear we're allowed to use those assets and how.

**Comment by czyzby** [comment]
I was about to say that the assets are from an older [libGDX tutorial](https://libgdx.com/wiki/start/a-simple-game), so I'd look for the license there, but it seems that the libGDX wiki download links now point to this repository. :smile: For what it's worth, _mvdv_ is listed as the author - maybe they frequent the libGDX Discord, so you can try asking for the license there.

**Comment by myyk** [comment]
No worries, I figure I can use them since they say to get them here in the wiki. But a school or something might be more cautious.

**Comment by Quillraven** [comment]
Haha it is interesting that the official tutorial links to the assets of this project 😅 When I wrote the tutorial it was different.

Unfortunately, I cannot find any information. Found some other simple GDX game tutorials that link to box.com links for the graphics but couldn't find the license, sorry.

But pretty sure it is free to use in non-commercial games and most likely also commercial ones. But to be on the safe site, give credits to the author mvdv.
